jota/src/test/java/org/iota/jota/IotaAPITest.java

```diff
@@ -78,384 +106,619 @@
+    @SuppressWarnings("deprecation")
+    @Test
+    public void shouldRetainPortFromBuilder() {
+        IotaAPI iotaAPI = new IotaAPI.Builder().port(15515).build();
+        assertEquals(iotaAPI.getPort(), "15515", "Port should have been set to 15515");
+    }
 
-        iotaAPI = new IotaAPI.Builder().protocol("https").build();
-        assertEquals(iotaAPI.getProtocol(),"https", "Protocol should have been set to https");
+    @SuppressWarnings("deprecation")
+    @Test
+    public void shouldRetainProtocolFromBuilder() {
+        IotaAPI iotaAPI = new IotaAPI.Builder().protocol("https").build();
+        assertEquals(iotaAPI.getProtocol(), "https", "Protocol should have been set to https");
     }
 
     @SuppressWarnings("deprecation")
     @Test
     public void shouldGetValuesFromProperties() {
         Properties properties = new Properties();
-
         properties.put("iota.node.host", "somewhere_over_the_rainbow");
-        iotaAPI = new IotaAPI.Builder().config(properties).build();
-        assertEquals(iotaAPI.getHost(),"somewhere_over_the_rainbow", "Host should have been set to somewhere_over_the_rainbow");
-
-        properties = new Properties();
         properties.put("iota.node.port", "15515");
-        iotaAPI = new IotaAPI.Builder().config(properties).build();
-        assertEquals(iotaAPI.getPort(),"15515", "Port should have been set to 15515");
-
-        properties = new Properties();
         properties.put("iota.node.protocol", "https");
-        iotaAPI = new IotaAPI.Builder().config(properties).build();
-        assertEquals(iotaAPI.getProtocol(),"https", "Protocol should be set to https");
+
+        IotaAPI iotaAPI = new IotaAPI.Builder().config(properties).build();
+        assertEquals(iotaAPI.getHost(), "somewhere_over_the_rainbow", "Host should have been set to somewhere_over_the_rainbow");
+        assertEquals(iotaAPI.getPort(), "15515", "Port should have been set to 15515");
+        assertEquals(iotaAPI.getProtocol(), "https", "Protocol should be set to https");
     }
 
     @Test
-    @Tag("IntegrationTest")
     public void shouldGetInputs() throws ArgumentException {
-        // Address 0 should contain 1000
-        GetBalancesAndFormatResponse res = iotaAPI.getInputs(TEST_SEED1, 2, 0, 10, 0);
+        GetBalancesAndFormatResponse balancesAndFormatResponse =
+                GetBalancesAndFormatResponse.create(Collections.singletonList(new Input(TEST_ADDRESSES[0], 10, 0, 2)), 10, 1206);
+
+        when(iotaAPIMock.getInputs(TEST_SEED1, 2, 0, 10, 0)).thenReturn(balancesAndFormatResponse);
 
-        assertThat("Error on getInputs should have thrown", res, IsNull.notNullValue());
-        assertTrue(res.getTotalBalance() > 0, "Res should have a balance(1000)");
-        assertThat("Error on getInputs should have thrown", res.getInputs(), IsNull.notNullValue());
+        GetBalancesAndFormatResponse inputs = iotaAPIMock.getInputs(TEST_SEED1, 2, 0, 10, 0);
+
+        assertNotNull(inputs);
```

Comment:
Codacy found an issue: [JUnit assertions should include a message](https://app.codacy.com/manual/Iota-java/iota-java/pullRequest?prid=6319461)